rules/automate-reply-messages/rule.md

```diff
@@ -0,0 +1,79 @@
+---
+type: rule
+title: LinkedIn - Do you know how to automate replies to company messages?
+uri: automate-reply-messages
```

Comment:
change to linkedin-automate-reply-messages

Reply:
I will do after re our chat about updating the folder size